Earthfile

```diff
@@ -130,3 +130,34 @@ test-integration:
 test:
   BUILD +test-unit
   BUILD +test-integration
+
+# -----------------------Building-----------------------
+
+build:
+  ARG GOARCH=arm64
+  ENV CGO_ENABLED=0 
+  ENV GOARCH=$GOARCH
```

Comment:
All ARGs are available as ENVs in the container, we can safely remove ENV GOARCH.

```suggestion
```

Reply:
yeah, I was curios about that, could not find information